test/views/ViewArrivalsSearchResultsSpec.scala

```diff
@@ -66,16 +66,9 @@ class ViewArrivalsSpec extends MovementsTableViewBehaviours("viewArrivals.njk")
 
   behave like pageWithLink(
```

Comment:
We could create a behaviour for the new search macro we have made, this would make creating view tests much easier if we need to add more in  the future

Reply:
Done that now, although we could also possible add a test that the form submits to the correct url